src/components/calcite-action-bar/calcite-action-bar.tsx

```diff
@@ -58,9 +58,7 @@ export class CalciteActionBar {
   @Prop() textCollapse = "Collapse";
 
   /**
-   * Arrangement of the component. Leading and trailing are different depending on if the direction is LTR or RTL. For example, "leading"
-   * in a LTR app will appear on the left.
-   * @deprecated()
+   * @deprecated Use `position` instead.
```

Comment:
WDYT of still keeping the old doc while calling out the deprecation notice?

```
/** 
 * @deprecated since <version> - use <alternative> instead.
 * 
 * <prop doc>
 */
```

cc @kat10140

Reply:
👍🏼